omnigibson/systems/system_base.py

```diff
@@ -1182,6 +1182,9 @@ def customize_mat(mat):
 
         # Generate the requested system
         system_cls = "".join([st.capitalize() for st in system_type.split("_")])
+        # MicroPhysicalParticle --> Granular
+        if system_cls == "MicroPhysicalParticle":
+            system_cls = "Granular"
```

Comment:
nit: another slightly hacky thing, can we do it a cleaner way?

Reply:
fixed